src/pyeval.jl

```diff
@@ -202,32 +198,37 @@ pasted into the Python code.   This allows you to evaluate code
 where the code itself is generated by a Julia expression.
 """
 macro py_str(code, options...)
+    m = :(pynamespace($__module__))
+    fname = String(__source__.file)
+    esc(:(PyCall.@_py_str($m, $m, $fname, $code, $(options...))))
```

Comment:
```suggestion
    esc(:($PyCall.@_py_str($m, $m, $fname, $code, $(options...))))
```

I think you need this?

I think we better test `py""` with something like

```julia
baremodule TestPyStr
using PyCall: @py_str
pystr(x) = py"$$x"
end
@test TestPyStr.pystr("1+1") == 2
```